### PR TITLE
add univariate analysis to walk-through.

### PR DESCRIPTION
**Pull request by zkamvar**
This will address #19

### CODE REVIEW

File: content/training/Walk-through/univariate.Rmd [hunk 1]
```diff
@@ -138,7 +138,7 @@ tab_linelist(age_group, sex, strata = case_def, na.rm = FALSE) %>%
   ## run chi-sq test
```
Comment (aspina7) [inline content/training/Walk-through/univariate.Rmd:132]: sorry this is the first place i could comment but in line 103 - it calls a vector of chars a dataframe... just for consistency

Comment (aspina7) [reply content/training/Walk-through/univariate.Rmd:132]: and again in line 107  - and needs a second hashtag

**Review by aspina7 — APPROVED** [review]

File: content/training/Walk-through/univariate.Rmd (hunk 1 above)
Comment (zkamvar) [reply content/training/Walk-through/univariate.Rmd:132]: I think I might have fixed these?

Comment (epiamsterdam) [reply content/training/Walk-through/univariate.Rmd:132]: sorry. my bad.

Comment (epiamsterdam) [reply content/training/Walk-through/univariate.Rmd:132]: they are still there Zhian

Comment (zkamvar) [reply content/training/Walk-through/univariate.Rmd:132]: Just checked. The only instances that use "data frame" are indeed data frames

Comment (epiamsterdam) [reply content/training/Walk-through/univariate.Rmd:132]: OK. Good.

**Pull request merged**